src/actions/index.js

```diff
@@ -9,10 +9,16 @@ export const DECREMENT = 'DECREMENT';
 // logic of updating the central Redux store. That
 // is left to the reducer(s).
 
-export const increment = () => {
+export const increment = count => {
   // Fill in this function
+  return{
+    type:INCREMENT
+  }
 };
 
-export const decrement = () => {
+export const decrement = count => {
```

Comment:
Doesn't break the code, but no need for `count` here if you're not returning a payload